### ISSUE TITLE
error Could not parse row {...

### ISSUE THREAD

**Issue by jikkujose** [diasdavid/ipscend#21]
Can't publish, keep getting error: `error Could not parse row {...`

What I did:
1. Created a simple html page in directory: _public_.
2. `ipscend init` & chose default location as _public_.
3. Previewed the HTML page via: `ipscend preview`, it worked as expected.
4. Tried to publish: `ipscend publish` and got the error specified.

Any ideas?

**Comment by daviddias** [comment]
@JikkuJose were you running an IPFS node in your machine? This tells me that we should have better error messages

**Comment by jikkujose** [comment]
@diasdavid yes, I ran the daemon while I was pushing it. I don't generally run it in the background all the time, since I am on a laptop.

**Comment by hackergrrl** [comment]
@JikkuJose What version of ipfs are you running? (`ipfs version`)  I saw the same thing when I tried to use `ipscend` with `go-ipfs@0.4.0`.

**Comment by ghost** [comment]
I get the same error when I try to use the js-ipfs-api to add something. The issue was closed at the go-ipfs repo, however.
https://github.com/ipfs/go-ipfs/issues/2109
I'm using:
ipfs version 0.3.8-dev
js-ipfs-api version 2.10.1

**Comment by daviddias** [comment]
Hi @JikkuJose and @void4 , mind testing ipscend again, just released version 0.2.0 which should solve the incompatibility issue. Also now you can do `ipscend ipfs start` to spawn a daemon that is compatible with the js-ipfs-api that ipscend is using.

**Comment by tinybike** [comment]
I also get this problem when using js-ipfs-api, which has made js-ipfs-api unusable for me in Nodejs.

**Comment by daviddias** [comment]
@tinybike which version (Node.js, js-ipfs-api and IPFS)?

**Comment by tinybike** [comment]
Node.js v4.2.4, js-ipfs-api v2.13.1 (from npm), IPFS v0.4.0-dev.  I saw a comment that feat/files-api had a fix for this issue so I tried this branch out, but it did not help me.

Temporary fix: I downgraded IPFS to v0.3.11 and js-ipfs-api works again.

**Comment by daviddias** [comment]
@tinybike yeah, ipscend is not 0.4.0 compatible (https://github.com/diasdavid/ipscend/pull/41) due to differences in IPFS HTTP API

**Comment by daviddias** [comment]
Hi all! It has been a long time. I've just updated the dependencies and ran some quick tests. It worked well, let me know if it works for you too.

To be 100% sure it won't break in the future I'll need to write some tests. I plan to spend some time on this once I get a bit of free time (which by the looks might be the end of June). Thank you for being patient :)

**Comment by daviddias** [comment]
Hi folks! I'm no longer maintaining this module. The good news is that there is one way better on the block, check the deprecation notice -- https://github.com/ipfs-shipyard/ipscend/blob/master/README.md#deprecation-notice -- to find a url to it and a tutorial on how to use it!
